widget/tabbed_misc/custom_tasklist.lua

```diff
@@ -0,0 +1,58 @@
+local wibox = require('wibox')
+local awful = require('awful')
+local gears = require('gears')
+local beautiful = require('beautiful')
+local dpi = require("beautiful.xresources").apply_dpi
+
+local function tabobj_support(self, c, index, clients)
+	-- Self is the background widget in this context
+	if not c.bling_tabbed then
+		return
+	end
+
+	local group = c.bling_tabbed 
+
+	-- Single item tabbed group's dont get special rendering
+	if #group.clients > 0 then
```

Comment:
Shouldn't it be `> 1` here? Because the client itself is also in that list afaik. I experienced that when I tabbed two windows and untabbed them again, the taskbar icon wouldn't change back for one of them. Might be related to that